docs/application/native/guides/multimedia/image-edit.md

```diff
@@ -321,111 +437,67 @@ To decode a JPEG, PNG, GIF, or BMP image:
 
 To encode a raw image:
 
-1. Create an encoding handle using the `image_util_encode_create()` function:
+1. Create an encoding handle using `image_util_encode_create()`:
 
    ```
    image_util_type_e encoder_type = IMAGE_UTIL_JPEG;
    image_util_encode_h encode_h = NULL;
    ret = image_util_encode_create(encoder_type, &encode_h);
    ```
 
-2. Set the image to the input buffer using the `image_util_encode_set_input_buffer()` function:
-
-   ```
-   ret = image_util_encode_set_input_buffer(encode_h, buffer);
-   ```
-
-3. Set the width, height, and color space of the input buffer using the `image_util_encode_set_resolution()` and `image_util_encode_set_colorspace()` functions:
+2. Additionally, you can set the JPEG quality or PNG compression using `image_util_encode_set_quality()` or `image_util_encode_set_png_compression()`:
 
    ```
-   ret = image_util_encode_set_resolution(encode_h, width, height);
-   ret = image_util_encode_set_colorspace(encode_h, IMAGE_UTIL_COLORSPACE_RGBA8888);
+   ret = image_util_encode_set_jpeg_downscale(decode_h, IMAGE_UTIL_DOWNSCALE_1_1);
```

Comment:
Would you please check `image_util_encode_set_jpeg_downscale`? I can't find it in 5.5 native API.

Reply:
`image_util_encode_set_jpeg_downscale` should be changed to `image_util_encode_set_quality` in the sample code.
I will replace the sample code.